packages/compiler/core/cli/args.ts

```diff
@@ -0,0 +1,162 @@
+import { expandConfigVariables } from "../../config/config-interpolation.js";
+import { loadCadlConfigForPath, validateConfigPathsAbsolute } from "../../config/config-loader.js";
+import { CadlConfig } from "../../config/types.js";
+import { createDiagnosticCollector } from "../index.js";
+import { CompilerOptions } from "../options.js";
+import { resolvePath } from "../path-utils.js";
+import { CompilerHost, Diagnostic } from "../types.js";
+import { omitUndefined } from "../util.js";
+
+export interface CompileCliArgs {
+  "output-dir"?: string;
+  "output-path"?: string;
```

Comment:
what is output-path?  This seems confusing alongside output-dir

Reply:
Is this just for backward compatibility?

Reply:
Yeah this was just for backward compatibility, it is deprecated.